requirements.txt

```diff
@@ -0,0 +1,2 @@
+requests==2.12.4
```

Comment:
Stripe seems to requires it internally, or at least think it does https://github.com/stripe/stripe-python/blob/master/setup.py